### ISSUE TITLE
[bug] Cannot install Bud 6.6.5

### ISSUE THREAD

**Issue by mensch** [roots/bud#1934]
### Agreement

- [X] This is not a duplicate of an existing issue
- [X] I have read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
- [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

### Describe the issue

I just created a new roots/sage theme, removed Tailwind, added some other bud packages and bumped the version number to 6.6.5. Running `yarn install` the installation now hangs on upgrading `@roots/bud`. I can install 6.6.3 just fine, but 6.6.5 somehow gets into an infinite loop, slowly eating up memory. When I terminate the command by pressing Ctrl+C this is the output on the terminal:

```➤ YN0007: │ @roots/bud@npm:6.6.5 [ed0cd] must be built because it never has been before or the last one failed
^C➤ YN0009: │ @roots/bud@npm:6.6.5 [ed0cd] couldn't be built successfully (exit code 129, logs can be found here: /private/var/folders/4f/h19r8j2j25xg29kdytj48_xh0000gn/T/xfs-a3b76c02/build.log)
```

The log contains the following information:

```# This file contains the result of Yarn building a package (@roots/bud@virtual:ed0cd50c76f5743d05eaecadcc817c2cfe7a58f7e175812e2fc4837499c46e624300588f2e7de14acc3761a434ba748194828ea43f5d41a921e581d78871f25e#npm:6.6.5)
# Script name: postinstall

updating browserslist database
/Users/path/to/website/web/app/themes/theme-name
Latest version:     1.0.30001439

Updating caniuse-lite version
$ yarn up -R caniuse-lite
```

The devDependencies of my package.json are as follows:
```
  "devDependencies": {
    "@roots/bud": "6.6.5",
    "@roots/bud-eslint": "6.6.5",
    "@roots/bud-prettier": "6.6.5",
    "@roots/bud-sass": "6.6.5",
    "@roots/bud-stylelint": "6.6.5",
    "@roots/eslint-config": "6.6.5",
    "@roots/sage": "6.6.5"
  },
```

### Expected Behavior

Installation of 6.6.5.

### Actual Behavior

The installation hangs on installing @roots/bud.

### Steps To Reproduce

1. Add a new roots/sage theme
2. Remove all references of Tailwind
3. Bump packages to 6.6.5
4. Run `yarn install`

### version

6.6.3

### What package manager are you using?

yarn

### version

3.3.0

**Comment by retlehs** [comment]
I'm unable to reproduce this. Please create a support topic on https://discourse.roots.io/ if you can't resolve this issue.

**Comment by mensch** [comment]
I'd be happy to open a support topic, but just to be sure, are you also unable to reproduce the issue when installing packages after setting yarn's version to berry (`yarn set version berry`)? I can install all dependencies just fine with the non-berry version of yarn, but it consistently fails after setting `yarn set version berry`.

**Comment by retlehs** [comment]
@mensch Sorry about the quick close and thanks for checking — I'm now seeing the same behavior with Berry & the deps you provided

**Comment by kellymears** [comment]
```sh
^C➤ YN0009: │ @roots/bud@npm:6.6.5 [ed0cd] couldn't be built successfully (exit code 129, logs can be found here: /private/var/folders/4f/h19r8j2j25xg29kdytj48_xh0000gn/T/xfs-a3b76c02/build.log)```
```
can you give me the output of this `build.log`? i'll see if i can reproduce in the meantime.

**Comment by kellymears** [comment]
Never mind, I could reproduce it. It does seem like you can use the project even after it fails (which makes sense as we're only updating browserslist; bud.js is released pre-compiled).

Can you run `browserslist` at all, out of curiosity?

```sh
# with yarn dlx
yarn dlx browserslist@latest --update-db
# or npm
npx browserslist@latest --update-db
```

Neither of these work for me with `yarn berry` 🤔.

This is not a critical script, in the meantime you can easily prevent it from running and fix any failures in CI or dev images by adding `dependenciesMeta` to `package.json`:

```json
{
  "dependenciesMeta": {
    "@roots/bud": {
      "built": false
    }
  }
}
```

**Comment by mensch** [comment]
I added the log in my original ticket, but Github seems to have mangled the formatting (fixed it just now). The command hung on `browserlist` for me as well. When I run `yarn dlx browserslist@latest --update-db` in my current setup (all bud packages are on 6.5.3 there) it seems to complete properly:

```
➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed in 0s 778ms
➤ YN0000: ┌ Fetch step
➤ YN0000: └ Completed
➤ YN0000: ┌ Link step
➤ YN0000: └ Completed in 0s 672ms
➤ YN0000: Done in 1s 618ms

Latest version:     1.0.30001439
Updating caniuse-lite version
$ yarn up -R caniuse-lite
caniuse-lite has been successfully updated

No target browser changes
```

**Comment by mensch** [comment]
This seems to be resolved with the release of 6.6.6, I've just successfully migrated two existing projects to the latest version of bud. Also created a new theme and bumped the packages to 6.6.6, set the version to `berry` and the installed all packages without a hitch.
